### PR TITLE
ci: releasing source code

### PR DESCRIPTION
**Pull request by sandipndev**
This PR adds Source code releasing. It uses a script from `meta-release` [here](https://github.com/GaloyMoney/meta-release/blob/main/ci/tasks/prep-release-src.sh).

### CODE REVIEW

**Comment by sandipndev** [comment]
I think Prettier should not be checking `pipeline.yml` the way it's doing it now coz it's repiped via `ytt`.

File: ci/pipeline.yml [hunk 1]
```diff
@@ -59,18 +72,64 @@ jobs:
       - put: edge-image
         params:
           image: image/image.tar
+  
+  - name: release
+    plan:
+    - in_parallel:
+      - get: repo
+        trigger: true
```
Comment (bodymindarts) [inline ci/pipeline.yml:80]: Its better if only 1 triggers I think. Repo or image.

File: ci/pipeline.yml [hunk 2]
```diff
@@ -59,18 +72,64 @@ jobs:
...
+        - name: version
+        - name: charts-repo
+        outputs:
+        - name: version
+        - name: artifacts
+        params:
+          CHART: admin-panel
+        run:
+          path: meta-release/ci/tasks/prep-release-src.sh
+    - put: edge-image
```
Comment (bodymindarts) [inline ci/pipeline.yml:107]: this should not go out to `edge-image` - perhaps `versioned-image` or something like that.

**Review by bodymindarts — APPROVED** [review]

File: ci/pipeline.yml (hunk 2 above)
Comment (sandipndev) [reply ci/pipeline.yml]: yeah, agreed, I have added it!

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**